airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -1808,6 +1808,8 @@ def __init__(
         gcp_conn_id: str = "google_cloud_default",
         impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
         asynchronous: bool = False,
+        hook: Optional[DataprocHook] = None,
+        job_id: str = '',
```

Comment:
I think we always use such notation.
```suggestion
        job_id: str,
```

Reply:
@michalslowikowski00  > I think we always use such notation.

I think such change will raise `SyntaxError: non-default argument follows default argument`, since `job_id` is optional.

What about the following notation?
```
job_id: Optional[str] = None,
```

But I got mypy error with the above, becouse `job_id` might be `None` when method `cancel_job` on `hook` is called.